distributed/client.py

```diff
@@ -4601,9 +4601,6 @@ def register_worker_plugin(self, plugin=None, name=None, nanny=None, **kwargs):
             If plugin has no name attribute a random name is used.
         nanny : bool, optional
             Whether to register the plugin with workers or nannies.
-        **kwargs : optional
-            If you pass a class as the plugin, instead of a class instance, then the
-            class will be instantiated with any extra keyword arguments.
```

Comment:
If we kill this doc string we should be more specific for the `plugin` argument docs now, e.g.

`The initialized plugin to register.`